### DISCUSSION TITLE
插件运行异常sherpaonnx.dll

### DISCUSSION THREAD

**Discussion by tian-ruo-you-qing** [jxlpzqc/TMSpeech#43] [Q&A] [answered]
(TMSpeech.Recognizer.SherpaOnnx.dll):
Cannot find model file:models\encoder.onnx
Current working directory:D:\Program Files\T

**Comment by jxlpzqc** [comment, marked as answer]
请在配置窗口中指定模型（可以指定自定义模型，也可以在《资源》页面下载模型），请参考 #42
